src/components/DoctorsPage/DoctorsCard/Schedule/Schedule.tsx

```diff
@@ -32,43 +45,45 @@ const reducer = (state: IScheduleInitialState, action: ActionType): IScheduleIni
 };
 
 const Schedule: React.FC<IScheduleProps> = ({ workTimeData }) => {
-  const [stateSchedule, dispatch] = useReducer(reducer, initialState);
+  const [stateSchedule, dispatchLock] = useReducer(reducer, initialState);
+  const [disabledButton, setDisabledButton] = useState<boolean>(false);
```

Comment:
what for having two local states in the same component? Can it be all solved via reducer?

Reply:
in all files if there are multiple local states i added useReducer